app/api/news/news.controller.js

```diff
@@ -0,0 +1,67 @@
+const Sequelize = require("sequelize");
+const config = require("../../../config");
+const models = require("../../models");
+const utils = require("../../utils/express");
+const mediaUtils = require("../../utils/media");
+
+const Op = Sequelize.Op;
+
+exports.getList = utils.route(async (req, res) => {
+  const lang = req.query.lang || (req.getLocale() || "en");
+
+  const offset = req.query.offset ? parseInt(req.query.offset) : 0;
+  const limit = req.query.limit ? parseInt(req.query.limit) : 10;
+
+  const news = await models.news.findAndCountAll({
+    limit,
+    offset,
+    order: [["published_at", "DESC"]],
+    where: {
+      published_at: {
+        [Op.lte]: new Date(),
+      },
+    },
+  });
+
+  const newsPaginated = {
+    data: {
+      news: news.rows.map((news) => {
+        return {
+          id: news.id,
+          title: news.title[lang],
+          description: news.description[lang],
+          description_preview: news.description_preview[lang],
+          img_url: news.img_url,
+          img_alt: news.img_alt ? news.img_alt[lang] : null,
```

Comment:
should we check that if there are no values for the language queried for that news, we default to english?